app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    list_words = command.split()
```

Comment:
User can enter commands like: 
`cp file1.txt` - not enough parameters
`mv file1.txt file2.txt` - wrong command
`cp file1.txt file2.txt file3.txt` - too much parameters

Add something to check these cases as well.